eva-accession-release-automation/run_release_in_embassy/create_release_properties_file.py

```diff
@@ -72,7 +69,7 @@ def create_release_properties_file_for_assembly(private_config_xml_file, profile
         parameters.assemblyAccession={assembly_accession}
         parameters.assemblyReportUrl={report_path}
         parameters.chunkSize=1000
-        parameters.contigNaming=SEQUENCE_NAME
+        parameters.contigNaming=INSDC
```

Comment:
That's right, this is so the release pipeline outputs vcfs using INSDC accessions rather than genbank sequence names, and the contig name converter can only convert from INSDC to ENA sequence names... it's a bit convoluted but I guess it's because the release pipeline is written to only use the NCBI assembly reports...

Reply:
The Sequence converter doesn't use the contig name in each record to convert but [uses the INSDC accession](https://github.com/EBIvariation/contig-name-converter/blob/c830fa1fdc7073f2c25ac9dcbff9268ba3384718/contig_name_converter/convert_vcf_file.py#L26) added to the VCF header. 
Changing to INSDC here means that we will have to change all the VCF records to get the ENA sequence name where using Genbank will be in many case similar to ENA which will result in [no change](https://github.com/EBIvariation/contig-name-converter/blob/c830fa1fdc7073f2c25ac9dcbff9268ba3384718/contig_name_converter/convert_vcf_file.py#L34).

Reply:
Okay, I need to check this, I think I was seeing errors from querying contig alias for things like `chrA1` and making this change fixed it.  But what you say makes sense so there might be something else going on.